instrumentation/net/http/otelhttp/config.go

```diff
@@ -206,3 +209,17 @@ func WithServerName(server string) Option {
 		c.ServerName = server
 	})
 }
+
+// WithRequestAttributeGetter extracts additional attributes from the request.
+func WithRequestAttributeGetter(fn func(req *http.Request) []attribute.KeyValue) Option {
+	return optionFunc(func(o *config) {
+		o.GetRequestAttributes = fn
+	})
+}
+
+// WithResponseAttributeGetter extracts additional attributes from the response.
+func WithResponseAttributeGetter(fn func(req *http.Response) []attribute.KeyValue) Option {
+	return optionFunc(func(o *config) {
+		o.GetResponseAttributes = fn
+	})
+}
```

Comment:
People may NOT want to add always the same attributes to spans and metrics. While for spans it is not a problem for metrics it leads to high cardinality.

I strongly suggest to remove this functionality from this PR. We can create a different solution in a separate PR.

Reply:
removed.